force-app/main/default/classes/AccountTriggerHandler.cls

```diff
@@ -0,0 +1,13 @@
+public with sharing class AccountTriggerHandler extends TriggerHandler{
+    protected override void processBeforeInsert() {
+        if (!TriggerBypassUtility.shouldBypass(UserInfo.getUserId())) {
```

Comment:
[MAJOR]
Przenieś to do Trigger Handlera

Reply:
Dodaj warunek w trigger handlerze

Reply:
Dodaj permission set lub profil admina przypisz do klas i wrzuć do repo